x-pack/packages/kbn-ecs-data-quality-dashboard/impl/data_quality/mock/uallowed_values/mock_uallowed_values.ts

```diff
@@ -0,0 +1,122 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+export const mockUnallowedValuesResponse = [
```

Comment:
nit: missed the 'n' in `unallowed` for the name of this file :) and file path :)